src/server/scripts/Spells/spell_rogue.cpp

```diff
@@ -778,7 +854,8 @@ void AddSC_rogue_spell_scripts()
     RegisterSpellScript(spell_rog_nerves_of_steel);
     RegisterSpellScript(spell_rog_preparation);
     RegisterSpellScript(spell_rog_prey_on_the_weak);
-    RegisterSpellScript(spell_rog_rupture);
+    new spell_rog_rupture();
+    new spell_rog_glyph_of_backstab_triggered();
```

Comment:
I dont think you need to use old registry here. use RegisterSpellScript, or if they are tied together with an aurascript use RegisterSpellAndAuraScriptPair()